other-docs/guides/upgrading/v11.md

```diff
@@ -49,6 +49,14 @@ ElasticPress has been updated to version 4.0 and will require a full reindex aft
 
 [HSTS can be switched off via the Altis config by following the documentation here](docs://security/browser.md#strict-transport-policy).
 
+## PHP 8 Readiness
```

Comment:
Maybe link your above note to this?

Reply:
I think I might just link the note in the compatibility table section to the compatibility check stuff in the same document. This is the upgrade guide so it's not going to make sense to link to what will become an outdated guide when the next version comes out.

Reply:
I only suggest it because this text was clearer than the readme note. :)

Reply:
I've updated the wording - was easier in the upgrade guide because it's time specific while the upgrading PHP guide should be a bit more evergreen.